### PR TITLE
docs: Added documentation for latex component

### PR DESCRIPTION
**Pull request by jfmcquade**
PR Checklist

- [x] - Latest `master` branch merged
- [x] - PR title descriptive (can be used in release notes)

## Description

Adds a page to the documentation explaining the usage of the latex component

## Screenshots/Videos
Screenshot 2022-04-21 at 11 21 27

### CODE REVIEW

File: documentation/docs/components/latex.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+You can write text that contains expressions inline like this: $\sum_{i=1}^n(x_i^2 - \overline{x}^2)$, by using the format \$expression\$.
+You also can write expressions in display mode as follows: $$\sum_{i=1}^n(x_i^2 - \overline{x}^2)$$
+This is achieved by using a double \$ symbol, like \$\$expression\$\$.
+You can escape the \$ by using \\$ in the text input.
+```  
+
+![](images/latex.png)
+
+[Google Sheet Demo](https://docs.google.com/spreadsheets/d/1qPNAFQbrEQX8ZahmTr3LGhx3uoaooQJqVOAF2HHwmcg/)   
+[Live Preview Demo](https://plh-teens-app1.web.app/template/comp_latex)
```
Comment (EsmeeIDEMS) [inline documentation/docs/components/latex.md:21]:
```suggestion
[Live Preview Demo](https://plh-global.web.app/template/comp_latex)
```

Comment (EsmeeIDEMS) [reply documentation/docs/components/latex.md:21]: We're no longer keeping the teens-app1 link up to date so it's best to use this master preview link instead.

**Review by EsmeeIDEMS — APPROVED** [review]
Looks good.

**Review by chrismclarke — APPROVED** [review]
Looks good to me, glad you were able to get things up and running. Thanks for updating the outdated links also.

We don't currently have a good system to preview docs changes, so thanks for adding the screenshot. Another easy way to generate them (for longer pages) is to use chrome's inbuilt full size screenshot command which can be accessed from the command pallete (accessed from the 3-dot menu within the devleoper console (not the main 3-dot menu))

image

image

The output should include the entire scrollable page (assuming the scroll is at a top level and not nested within a child component). Here's an example output

127 0 0 1_8000_components_latex_

**Pull request merged**